rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/retrofit/RetrofitConfiguration.groovy

```diff
@@ -36,7 +36,7 @@ class RetrofitConfiguration {
   }
 
   @Bean LogLevel retrofitLogLevel() {
-    LogLevel.FULL
+    LogLevel.BASIC
```

Comment:
Please leave the log level as it was originally. Will be dropping this fairly shortly anyway, and will reduce the number of uncommitted changes in the meantime.